File: src/Alkoholspiel.Core/Entities/Joke.cs
namespace Alkoholspiel.Core.Entities;

public class Joke
{
    public Guid Id { get; }
    public string Author { get; }
    public string  Content { get; }

    public Joke(string author, string content)
    {
        Id = Guid.NewGuid();
        Author = author;
        Content = content;
    }

    public Joke(Guid id, string author, string content)
    {
        Id = id;
        Author = author;
        Content = content;
    }
}

File: core/src/main/kotlin/alkoholspiele/core/entities/Joke.kt
package alkoholspiele.core.entities

import java.util.UUID

class Joke {

    val id: UUID;
    val content: String;

    constructor(id: UUID, content: String) {
        this.id = id
        this.content = content
    }

    constructor(content: String) {
        this.id = UUID.randomUUID()
        this.content = content
    }


}